scripts/setupLocalProxy.sh

```diff
@@ -214,6 +214,7 @@ env() {
     echo export JC_IDLER_API_URL=http://localhost:${LOCAL_IDLER_PORT}
     echo export JC_F8TENANT_API_URL=http://localhost:${LOCAL_TENANT_PORT}
     echo export JC_OSO_CLUSTERS="'{\"https://api.free-stg.openshift.com/\": \"1b7d.free-stg.openshiftapps.com\"}'"
+    echo export JC_INDEX_PATH=static/html/index.html
```

Comment:
@kishansagathiya could you please add to the commit message what this fixes? E.g.
>  Previously X fails with Y error if  `JC_INDEX_PATH` isn't set to index.html. This patch fixes it by setting the env var.

Reply:
@sthaha Have changed the commit message accordingly